scene/animation/animation_mixer.cpp

```diff
@@ -1073,15 +1076,18 @@ void AnimationMixer::_blend_calc_total_weight() {
 		real_t weight = ai.playback_info.weight;
 		Vector<real_t> track_weights = ai.playback_info.track_weights;
 		Vector<int> processed_indices;
```

Comment:
I suggest using LocalVector instead of caching the write pointer.

Reply:
See this PR https://github.com/godotengine/godot/pull/92257